tests/unit/retry/test_retry_unary.py

```diff
@@ -145,30 +109,25 @@ async def test_retry_target_warning_for_retry(utcnow, sleep):
         retry.retry_target(target, predicate, range(10), None)
 
     assert len(exc_info) == 2
-    assert str(exc_info[0].message) == retry._ASYNC_RETRY_WARNING
+    assert str(exc_info[0].message) == retry.retry_unary._ASYNC_RETRY_WARNING
     sleep.assert_not_called()
 
 
 @mock.patch("time.sleep", autospec=True)
-@mock.patch("google.api_core.datetime_helpers.utcnow", autospec=True)
-def test_retry_target_deadline_exceeded(utcnow, sleep):
+@mock.patch("time.monotonic", autospec=True)
+def test_retry_target_deadline_exceeded(monotonic, sleep):
     predicate = retry.if_exception_type(ValueError)
     exception = ValueError("meep")
     target = mock.Mock(side_effect=exception)
     # Setup the timeline so that the first call takes 5 seconds but the second
     # call takes 6, which puts the retry over the deadline.
-    utcnow.side_effect = [
-        # The first call to utcnow establishes the start of the timeline.
-        datetime.datetime.min,
-        datetime.datetime.min + datetime.timedelta(seconds=5),
-        datetime.datetime.min + datetime.timedelta(seconds=11),
-    ]
+    monotonic.side_effect = [0, 5, 11]
```

Comment:
Good point, but I also want to make sure we have some tests that use `deadline`, if that's supposed to be maintained as an alias. I added an extra `paramaterize` to check with both

But if we make this a breaking change, maybe we can drop deadline

Reply:
I like the parametrization idea! 

I think what you have is good. One could go a bit further but I think it would be less clear than what you have: 

```
@pytest.mark.parametrize("timeout_arg", ["deadline", "timeout"])
...

 kwargs = {timeout_arg: 10}
```